examples/armada.yaml

```diff
@@ -20,7 +20,7 @@ armada:
         source:
           type: git
           location: git://github.com/att-comdev/openstack-helm
-          subpath: common
+          subpath: helm-toolkit
```

Comment:
@alanmeadows sounds good to me. Will generate an issue for other common in the repo.